src/cdi_analytic/Pseudo_Line_Base.cc

```diff
@@ -249,7 +249,8 @@ double Pseudo_Line_Base::monoOpacity(double const x, double const T) const {
       Result += edge_factor_[i] * cube(nu0 / x);
     }
   }
-  if (Tpow_ != 0.0) {
+  if (!rtt_dsxx::soft_equiv(Tpow_, 0.0,
+                            std::numeric_limits<double>::epsilon())) {
```

Comment:
Why is the `numeric_limits` function used here instead of 1.0e-16 (or vice versa)?